### PR TITLE
Abdul/speech translator

### PR DESCRIPTION
**Pull request by v-abdrahman**
Created a folder for Speech Translator app & added all the project files for windows 10 Iot Enterprise.

Following changes are done:

1) file modified : SpeechTranslator.csproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.

2) file modified : README.md
    Adding windows 10 iot Enterprise in products.
    Updated the Windows Platform from IoT Core to IoT Enterprise
    Added the BIOS setting for UPBOARD Pin configuration
    Added the app *packages installing steps

### CODE REVIEW

**Review by rsameser — APPROVED** [review]

File: SpeechTranslator/CS/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  - Windows 10 
+  - Windows 10 IoT Enterprise
+description: Translate your speech to different languages.
+---
+
+# Build the Speech Translator Project
+
+## Component Lists
+- A Windows 10 IoT Enterprise Device (e.g. UP Board)
+- A headset [e.g. the Microsoft LifeChat-3000 Headset](http://www.microsoft.com/hardware/en-us/p/lifechat-lx-3000/JUG-00013)
```
Comment (neilsh-msft) [inline SpeechTranslator/CS/README.md]: The URL link no longer works.  This URL points at the device:  https://www.microsoft.com/accessories/en-ww/products/headsets/lifechat-lx-3000/jug-00013

File: SpeechTranslator/CS/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+# Build the Speech Translator Project
+
+## Component Lists
+- A Windows 10 IoT Enterprise Device (e.g. UP Board)
+- A headset [e.g. the Microsoft LifeChat-3000 Headset](http://www.microsoft.com/hardware/en-us/p/lifechat-lx-3000/JUG-00013) 
+- A mouse
+- A router connected to the Internet 
+- An ethernet cable
+- An HDMI monitor and cables 
+- A micro-SD card and reader
```
Comment (neilsh-msft) [inline SpeechTranslator/CS/README.md]: The UP Board doesn't appear to have an SD card interface.  Is the micro-SD card and reader still needed?

File: SpeechTranslator/CS/README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    3. Copy a language package from your desktop to your device.  
+        *Note: Language packages are available in this folder:* c:\Program Files (x86)\Windows Kits\10\MSPackages\retail\ **your-processor-type** \fre
+        
+        For example, to copy Spanish Mexican to an arm based device:
+        - **copy "c:\Program Files (x86)\Windows Kits\10\MSPackages\retail\amd64\fre\Microsoft-Windows-OneCore-Microsoft-SpeechData-es-MX-Package.cab" t:**                   
+    4. Unmap your local drive (e.g. net use /delete t:).
+    
+9. Apply the Speech Language Package to your device.  **(Required for Windows 10 IoT Builds 15063 or greater)**
+    
+    *Note: Your device should reboot after committing each update and a "spinning gears" screen will appear until the update completes.*
```
Comment (neilsh-msft) [inline SpeechTranslator/CS/README.md]: Does the spinning gears show up for iot enterprise?

File: SpeechTranslator/CS/ConstantParam.cs [hunk 4]
```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace SpeechTranslator
+{
+    class ConstantParam
+    {           
+        //Token Property
+        public static string subscriptionKey = "47fe38401a6e45699b1ff76c8d3681d1";
```
Comment (neilsh-msft) [inline SpeechTranslator/CS/ConstantParam.cs]: Revert this back to         

public static string subscriptionKey = "****REPLACE THIS WITH YOUR AZURE COGNITIVE SERVICES SUBSCRIPTION KEY****";

Don't commit private subscription keys.

**Review by neilsh-msft — CHANGES_REQUESTED** [review]

File: SpeechTranslator/CS/README.md (hunk 1 above)
Comment (v-abdrahman) [reply SpeechTranslator/CS/README.md]: Updated the working link for the headset.

File: SpeechTranslator/CS/README.md (hunk 2 above)
Comment (v-abdrahman) [reply SpeechTranslator/CS/README.md]: Removed the Micro-SD card and reader line

File: SpeechTranslator/CS/README.md (hunk 3 above)
Comment (v-abdrahman) [reply SpeechTranslator/CS/README.md]: Just Reboot happens, Removed the Note

File: SpeechTranslator/CS/ConstantParam.cs (hunk 4 above)
Comment (v-abdrahman) [reply SpeechTranslator/CS/ConstantParam.cs]: Done the changes.

**Pull request merged**